wrappers/python/tests/anoncreds/test_prover_get_claim_offers.py

```diff
@@ -1,57 +1,74 @@
 from indy.anoncreds import prover_get_claim_offers
 from indy.error import ErrorCode, IndyError
 
-from tests.utils import anoncreds
-
 import json
 import pytest
 
 
+# noinspection PyUnusedLocal
 @pytest.mark.asyncio
-async def test_prover_get_claim_offers_works_for_empty_filter(init_common_wallet):
-    claim_offers = json.loads(await prover_get_claim_offers(init_common_wallet[0], "{}"))
+async def test_prover_get_claim_offers_works_for_empty_filter(wallet_handle, prepopulated_wallet):
+    claim_offers = json.loads(
+        await prover_get_claim_offers(wallet_handle, "{}"))
+
     assert len(claim_offers) == 3
```

Comment:
Can we check that the returned claim offers match the expected result (as defined by API)?